src/OrleansRuntime/Core/InsideRuntimeClient.cs

```diff
@@ -473,6 +473,10 @@ private void SafeSendResponse(Message message, object resultObject)
             var methodInfo = typeof(Exception).GetMethod(
                 "PrepForRemoting",
                 BindingFlags.Instance | BindingFlags.NonPublic);
+
+            if (methodInfo == null)
+                return new Func<Exception, Exception>(exc => exc);
```

Comment:
We generally use cast delegates instead of using `new`.
Do you mind removing adding a comment here?